atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/IndexDefinition.java

```diff
@@ -139,15 +139,15 @@ public void componentFromColumn(String componentName, ValueType valueType, Strin
     }
 
     public void componentFromColumn(String componentName, ValueType valueType, ValueByteOrder valueByteOrder, String sourceColumnName, String codeToAccessValue) {
-        addComponent(IndexComponent.createFromColumn(new NameComponentDescription(componentName, valueType, valueByteOrder), sourceColumnName, codeToAccessValue));
+        addComponent(IndexComponent.createFromColumn(new NameComponentDescription.Builder().componentName(componentName).type(valueType).byteOrder(valueByteOrder).build(), sourceColumnName, codeToAccessValue));
     }
 
     /**
      * Allows multiple index rows when indexing by a cell with iterable values.
      * It doesn't support arbitrary protobuf structures - you need to be able to extract an iterable<valueType> using codeToAccessValue.
      */
     public void componentFromIterableColumn(String componentName, ValueType valueType, ValueByteOrder valueByteOrder, String sourceColumnName, String codeToAccessValue) {
-        addComponent(IndexComponent.createIterableFromColumn(new NameComponentDescription(componentName, valueType, valueByteOrder), sourceColumnName, codeToAccessValue));
+        addComponent(IndexComponent.createIterableFromColumn(new NameComponentDescription.Builder().componentName(componentName).type(valueType).byteOrder(valueByteOrder).build(), sourceColumnName, codeToAccessValue));
```

Comment:
nit: Seems like these lines are going to be >>120 chars. I realise we don't have baseline on this project, but might be nice to fix this.

Out of interest, was this generated with a `s/NameComponentDescription\(([^,]),([^,]),([^,])\)/NameComponentDescription.Builder\(\).componentName\(\1\).type\(\2\).byteOrder\(\3\).build\(\)/g`? 😄